app/gen-server/ApiServer.ts

```diff
@@ -574,6 +574,14 @@ export class ApiServer {
       });
     }));
 
+    // GET /api/users
+    // Get all users.
+    this._app.get('/api/users', expressWrap(async (req, res) => {
+      const users = await this._dbManager.getUsers();
+
+      return sendReply(req, res, users);
+    }));
```

Comment:
I wonder if that would not be a security issue, considering that it would expose the list of every users of an instance (their name and their email), no matter what access to their information the requester have.

Reply:
In my use case, I control all users in the instance, so there is no problem. You know more use cases, so you can tell if there is one that listing all users might be a security issue. Do you think it would be better to list only users who have access to the current site?